atlas

```diff
@@ -12,13 +12,12 @@ parser_definition() {
 }
 
 parser_definition_pull() {
-	setup   REST help:usage_pull -- \
-		"Usage: atlas pull [options...] [arguments...]"
+	setup   PULL_REST help:usage_pull -- \
```

Comment:
`eval "set -- $REST"` doesn't work when called a second time for `pull` argument.

I tried a dozen combination and it only worked after changing the variable name despite the official docs suggesting reusing `REST`.